src/components/FlightTable.tsx

```diff
@@ -0,0 +1,33 @@
+import { type Flight } from "../types";
+import { type ReactElement } from "react";
+
+const FlightTable = (props: { flights: Flight[] }): ReactElement => {
```

Comment:
Let's declare the properties of a component in a seperate file called `FlightTable.definitions.ts`. There we can make an interface such as:

```ts
export interface FlightTableProps {
   flights: Flight[]
}
```

and then in this line we can use it like this while destructuring at the same time:
```diff
- const FlightTable = (props: { flights: Flight[] }): ReactElement => {
+ const FlightTable = ({ flight }: FlightTableProps): ReactElement => {
```